components/social-icons/class-social-icons.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Social Icons component.
+ *
+ * @package Civil_First_Fleet
+ */
+
+namespace Civil_First_Fleet\Component;
+
+/**
+ * Social_Icons component class.
+ */
+class Social_Icons extends \Civil_First_Fleet\Component {
+
+  /**
+   * Unique component slug.
+   *
+   * @var string
+   */
+  public $slug = 'social-icons';
+
+  /**
+   * Default component settings.
+   *
+   * @return array Default settings.
+   */
+  public function default_settings() : array {
+    return array(
+      'context' => '',
+    );
```

Comment:
Short array syntax `[]` is usually preferred on our projects, but may vary project to project.